TheWarWithin/HunterBeastMastery.lua

```diff
@@ -2092,9 +2128,18 @@ spec:RegisterAbilities( {
         startsCombat = true,
 
         handler = function ()
-            applyBuff( "beast_cleave" )
+            applyBuff( "beast_cleave", 6 )
```

Comment:
Shouldn't need to specify the aura duration if beast_cleave's aura duration is set above.